### DISCUSSION TITLE
Errors in latest version

### DISCUSSION THREAD

**Discussion by akaravida** [Automattic/mongoose#10516] [Q&A] [answered]
[Hi](url) all!

 At the latest version I started getting tons of errors on build like mongodb has no exported member... Only working version is 10.19, the only that doesn't use @types/mongodb as a dependency. Do you have any solution? 

Also, when providing connectOptions, sslCA is not suppoted as buffer anymore, I inspected the file with declaration and the only type is the string?

![image](https://user-images.githubusercontent.com/87481910/127633214-8d9747d1-131f-44fa-99dd-7dcfa602ecfd.png)

**Comment by vkarpov15** [comment]
Please open an issue and follow the issue template. It looks like you might have mixed versions of Mongoose and MongoDB.

**Comment by hasezoey** [comment, marked as answer]
This seems like `mongodb` / `@types/mongodb` 4.x is installed, try downgrading & pinning to `3.6.x`
